src/Error/GraphQLException.php

```diff
@@ -181,6 +207,32 @@ public function getOriginalErrorMessage(): ?string
         return null !== $this->originalException ? $this->originalException->getMessage() : null;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function toArray(): array
+    {
+        $result = [
+            'message'    => $this->message,
+            'locations'  => $this->getLocationsAsArray(),
+            'path'       => $this->path,
+        ];
+
+        if (!empty($this->extensions)) {
```

Comment:
This should probably be `$this->extensions !== null`, an implementator might want extensions to be included even if it's an empty array

Reply:
Either that or extensions should always be an array, not `?array`

Reply:
True, I'll fix this.